### DISCUSSION TITLE
How to change mapping of default prompt?

### DISCUSSION THREAD

**Discussion by dhruvinsh** [olimorris/codecompanion.nvim#280] [Q&A]
Hi, Is it possible to change just the mapping of one of the default `prompt_library` ?

**Comment by olimorris** [comment]
The only way is to override it in your config like:

```lua
require("codecompanion").setup({
  prompt_library = {
    ["Custom Prompt"] = {
      opts = {
        mapping = "<LocalLeader>cp",
      },
    },
  },
})
```

**Reply by olimorris** [reply]
I'm planning on changing how the mappings work soon. I don't like the idea of automatically binding prompts to keys so I'll come up with a generic method for it
